src/btree/bt_split.c

```diff
@@ -856,6 +857,13 @@ __split_parent(WT_SESSION_IMPL *session, WT_REF *ref,
 			F_CLR_ATOMIC(parent, WT_PAGE_SPLITTING);
 			continue;
 		}
+	/*
+	 * If the no_wait flag is set, we should return ebusy.
```

Comment:
The indenting of this comment doesn't look right. We do 8-space unexpanded tabs and it should match the surrounding code.